google/validation.go

```diff
@@ -0,0 +1,24 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"regexp"
+)
+
+func validateGCPName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Spoke in person; I understand the differing signatures here. I'm fine with the code as is.